### PR TITLE
Add service-sires API - https://github.com/adewg/ICAR/issues/402

### PR DESCRIPTION
**Pull request by erwinspeybroeck**
https://github.com/adewg/ICAR/issues/402

### CODE REVIEW

File: url-schemes/exampleUrlScheme.json [hunk 1]
```diff
@@ -717,6 +717,38 @@
         }
       }
     },
+    "/service-sires": {
+      "get": {
+        "operationId": "get-service-sires for a given id",
```
Comment (erwinspeybroeck) [inline url-schemes/exampleUrlScheme.json]: "operationId": "get-service-sires for a given id" 
"operationId": "get-service-sires"

File: url-schemes/exampleUrlScheme.json [hunk 2]
```diff
@@ -717,6 +717,38 @@
         }
       }
     },
+    "/service-sires": {
+      "get": {
+        "operationId": "get-service-sires",
+        "summary": "Get the service sires matching your query parameters",
+        "description": "# Purpose\nProvides the service sires\n",
```
Comment (cookeac) [inline url-schemes/exampleUrlScheme.json]:
```suggestion
        "description": "# Purpose\nProvides the service sires.\nOnly animal fields relevant to service sires will be populated (location is not expected, for example).",
```

**Review by cookeac — CHANGES_REQUESTED** [review]
Looks good. In line with our discussion at the meeting, I'm suggesting a little more documentation so people know that only some fields will be populated.

File: url-schemes/exampleUrlScheme.json [hunk 3]
```diff
@@ -717,6 +717,38 @@
         }
       }
     },
+    "/service-sires": {
+      "get": {
+        "operationId": "get-service-sires",
+        "summary": "Get the service sires matching your query parameters",
```
Comment (erwinspeybroeck) [inline url-schemes/exampleUrlScheme.json]:
```suggestion
        "summary": "Get the service sires matching your query parameters - for now we only implement the query by some identifier, like lifetime number, ai-code, ... ",
```

**Comment by cookeac** [comment]
Resolves #402

**Pull request merged**